packages/jss/src/DomRenderer.js

```diff
@@ -130,17 +130,14 @@ const getHead = memoize((): HTMLElement => (document.querySelector('head'): any)
  * Find attached sheet with an index higher than the passed one.
  */
 function findHigherSheet(registry: Array<StyleSheet>, options: PriorityOptions): StyleSheet | null {
-  for (let i = 0; i < registry.length; i++) {
-    const sheet = registry[i]
-    if (
-      sheet.attached &&
-      sheet.options.index > options.index &&
-      sheet.options.insertionPoint === options.insertionPoint
-    ) {
-      return sheet
-    }
-  }
-  return null
+  return (
```

Comment:
this is mostly a cosmetic change that is also slower (on a micro level), we optimize everything in this lib to get the smallest bit of performance.

Reply:
So you mean this PR is not a good effect on JSS?

Reply:
I agree `for` is faster than `find`

Reply:
thank you